server/boot/01-frontend-server.js

```diff
@@ -0,0 +1,29 @@
+var Server = require('../frontend-server.js');
+var app = Server.app();
+
+
+// If the node_env is NOT set to production, run the webpackdev server
+// Uses the webpack.dev.config.js file for webpack configuration.
+if (process.env.NODE_ENV !== 'production') {
+
+	var webpack = require('webpack');
```

Comment:
Why does the webpack require have to be outside the if statement? It's only used for the dev-server? In production, it's just running a simple express app.